src/Test/L0/Util/WhichUtilL0.cs

```diff
@@ -212,5 +212,210 @@ public void WhichThrowsWhenSymlinkBroken()
             File.Delete(brokenSymlink);
             Environment.SetEnvironmentVariable(PathUtil.PathVariable, oldValue);
         }
+
+        [Fact]
+        [Trait("Level", "L0")]
+        [Trait("Category", "Common")]
+        public void UseWhich2FindGit()
```

Comment:
these tests are copy paste all existing tests but change `Which()` to `Which2()`